### ISSUE TITLE
[TF2] Allow F2P players to communicate on official servers 

### ISSUE THREAD

**Issue by Rongeno** [ValveSoftware/Source-1-Games#4177]
On the June 16th 2020 an update was released for TF2 that restricted certain new accounts from using chat in official matchmaking modes. This was done to mitigate the use of new and free accounts by hackers and cheating bots for abusive purposes, and while the intention behind it was good, to be honest, this change is incredibly bad and only realy hurts normal players. 
### Why this сhange is bad ###
- It realy confuses and upsets new players. I've seen a lot of people like these that just don't understand why they can't talk
- It makes TF2 Pay to Win since F2P players can't even call a medic or say that there is a spy which can be very important sometimes 
- Even if there would be a hacker that would spam a lot of bad things in chat, that hacker could be very easily muted by absolutely anyone. Because of that, this whole change to not let F2P players use communication tools seems kinda useless even for fighting against bots and hackers.
- Since new players can't talk it can make the game for them quite unenjoyable, which means there are more chances that new player will leave the game

It's been already 2 years since this change, a lot of exploits using the text chat were fixed and now there is not even a single reason for  F2P players not being able to communicate. A lot of people would be very thankful if that change could be reverted back.

**Comment by AzureAzel** [comment]
I agree with this. The ability to spam newlines in chat to completely disable chat for everyone has been fixed after this change, and as far as I know, placing newlines in your name is also fixed. (The latter being the reason why voice commands had to be disabled.)

Re-enabling chat for F2P players at this point would be more beneficial due to these issues being patched. Chat spammers will only make themselves a target for a kick at this point.

**Comment by Ashetf2** [comment]
> I agree with this as well, it's very unfortunate that F2P players can't even communicate with their team that there is a heavy or if they need Medic, not even with voice commands.
> 
> The bots have already found a way around this issue so I'd recommend changing it back. Even "clearing" the text chat doesn't even work anymore for bots.

They don't. Bot hosters just buy stolen premium Steam accounts

**Comment by luki1412** [comment]
Instead of removing F2P players' ability to chat, limit everyone to posting 1 message per 10s and the amount of characters they can post.

**Comment by condor00fr** [comment]
I agree, this change did more harm that good, besides the bots had managed to pass by this change at the time (not anymore trought) so wy wasn't it changed back?

**Comment by melon-mann** [comment]
Why hasn't this been reverted yet? It's as simple as removing a couple of lines of code, this doesn't do any harm as bots are already P2P accounts. This would make the game a lot more approachable to beginners, instead of paywalling a originally un-paywalled core feature.

It's been **2 years** Valve, please correct this.

**Comment by AmadeusTomboy** [comment]
I rarely find bots anymore, I think it's time to allow f2ps to talk

**Comment by Ashetf2** [comment]
Valve, with the Summer Update a lot of new players are trying the game. Keeping the restriction will just make them leave the game out of frustration. Please reconsider your position on this. Thanks.

**Comment by Ashetf2** [comment]
Since the bots have being gone for a while now, this may be a good moment to revert this decision.

**Comment by RyleiC** [comment]
Bots are seldom encountered nowadays, a revert for this would be in good order

**Comment by gaelcoral** [comment]
Fixed?
https://steamcommunity.com/games/TF2/announcements/detail/526468444896364569?snr=2___
![Image](https://github.com/user-attachments/assets/1e107563-5d20-45f2-ac66-efd4754041cb)

**Comment by Ashetf2** [comment]
Not really, or perhaps it's bugged.

**Comment by RyleiC** [comment]
With the last 2 patches have we fulfilled it now?

**Comment by kisak-valve** [comment]
Closing per the last handful of comments.

**Comment by vladisaev12** [comment]
> Fixed? https://steamcommunity.com/games/TF2/announcements/detail/526468444896364569?snr=2___ ![Image](https://github.com/user-attachments/assets/1e107563-5d20-45f2-ac66-efd4754041cb)

didn't do anything for like 90% of f2ps.
